travis-ci/terraform-config
# Update bin/travis-worker-verify-config to check for presence of librato credentials

## What is the problem that this PR is trying to fix?

Librato credentials seem missing on worker hosts, and there doesn't seem to be a check for these.

## What approach did you choose and why?

I'm attempting to check for these via `bin/travis-worker-verify-config`.

## How can you test this?

Good question! A comment in `bin/travis-worker-verify-config` says it can be executed directly, but it seems to expect an argument:
```
    def initialize(argv: ARGV)
      @cloud_config_string_base64 = argv.fetch(0)
      @prog = File.basename($PROGRAM_NAME)
    end
```
And I can't figure out what it wants :/

## What feedback would you like, if any?

Why aren't the Librato credentials being written to `/etc/default/travis-worker`?
